src/config/mod.rs

```diff
@@ -188,6 +254,10 @@ pub trait SourceConfig: core::fmt::Debug + Send + Sync {
     fn resources(&self) -> Vec<Resource> {
         Vec::new()
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
+    }
```

Comment:
I'm inclined to leave this default implementation out to force new component implementations to consider whether it should be true or false. I could see us forgetting to add an implementation to new sources that do support it.